server/src/main/kotlin/suwayomi/tachidesk/server/ServerConfig.kt

```diff
@@ -108,7 +109,8 @@ class ServerConfig(getConfig: () -> Config, val moduleName: String = SERVER_CONF
         }
 
         val sharedFlow = MutableSharedFlow<T>(extraBufferCapacity = 1, onBufferOverflow = BufferOverflow.DROP_OLDEST)
-        actualFlow.distinctUntilChanged().onEach { sharedFlow.emit(it) }.launchIn(mutableConfigValueScope)
+        actualFlow.distinctUntilChanged().onEach { mutableConfigValueScope.launch { sharedFlow.emit(it) } }
```

Comment:
I think changing it to mapLatest would be a better solution

Reply:
done